webpack/scenes/ContentViews/Details/ComponentContentViews/ContentViewComponents.js

```diff
@@ -216,8 +216,8 @@ const ContentViewComponents = ({ cvId, details }) => {
     </DropdownItem>,
   ];
 
-  const emptyContentTitle = __(`No content views belong to ${label}`);
-  const emptyContentBody = __('Please add some content views.');
+  const emptyContentTitle = __('No content views to add yet');
+  const emptyContentBody = __('Please create some component content views.');
```

Comment:
We don't use the term "component content views" any more..

```suggestion
  const emptyContentBody = __('Please create some content views.');
```